### ISSUE TITLE
GPU delegate: FILL and BROADCAST_TO ops not registered

### ISSUE THREAD

**Issue by sarmientoF** [google-ai-edge/LiteRT#6851]
## Problem

`tfl.fill` (opcode 94) and `tfl.broadcast_to` (opcode 126) are not registered in the GPU delegate's `NewOperationParser` switch. Both return `UnsupportedOperationParser` with generic "Operation is not supported."

Models exported via `litert-torch` produce these ops from `aten.full` / `aten.full_like` decompositions (e.g. mask creation). They block GPU delegation:

```
FILL: Operation is not supported.
BROADCAST_TO: Operation is not supported.
```

## Impact

~67 ops per model rejected. These are typically small constant-fill operations used for attention masks, zeros initialization, etc.

## Suggestion

Register `FILL` and/or `BROADCAST_TO` in `NewOperationParser` (`model_builder.cc`). Both are simple element-wise ops that Metal/OpenCL can handle trivially.

Alternatively, document recommended decomposition targets for GPU-compatible export — which ops SHOULD `aten.full` lower to?

## Environment

- macOS 15.x ARM64, Metal delegate
- Also affects OpenCL path (same `model_builder.cc`)

**Comment by sourcelite** [comment]
Hi again @sarmientoF,  

Thank you for this detailed feature request regarding "GPU delegate support for FILL and BROADCAST_TO". For us to evaluate best path forward - whether through registering these ops in delegate or adjusting the lowering behaviour in the conversion pipeline, could you please share a minimal pytorch code snippet or any specific model architecture configuration will generate the particular ```aten.full / aten.full_like``` masks during export ? Thanks alot.

**Comment by jl45G** [comment]
+ @terryheo for viz
